### ISSUE TITLE
Is this app still being developed?

### ISSUE THREAD

**Issue by ToughDBlue** [justdave/nwsweatheralertswidget#58]
Wasn't sure where to ask this. I was looking for an open source app that could send weather alerts by notifications and audible alerts and found this one. But it hasn't been updated since 2001. Are there any plans to update it?

**Comment by justdave** [comment]
Yes, I'm still planning to update it. Part of the problem is that Google made some changes to their policies surrounding background apps that basically necessitated a full rewrite of the app and that rewrite is still in progress. Also since it's a free app I'm having to fit it in around working on other things that I get paid for. The rewrite is about 2/3rds done. It will probably be mid-April before I get some time to work on it again.

**Comment by ToughDBlue** [comment]
Thank you for the quick reply. Thank you for your hard work and time making this app. I like that this is open source, no ads and no trackers. I'm excited to see this app in a new re-write. Currently there is only one app (that I know of) that just does weather alerts and that's "Severe Weather Alerts". It doesn't have ads but it's not open source and has 1 tracker. The location can be off by one county for some reason. The alerts can be setup to be audible, notification, or silent notification based on the specific weather alert. Hoping the new NWS Weather Alerts Widget has this as well. Thank you again for your time developing this app!
